gensrc/thrift/PaloInternalService.thrift

```diff
@@ -179,6 +179,8 @@ struct TQueryOptions {
   51: optional bool enable_new_shuffle_hash_method
 
   52: optional i32 be_exec_version = 0
+
+  53: optional bool enable_partitioned_hash_join = false
```

Comment:
please set default value to true. If not we could not make sure we passed Test Pipeline.